Specifications/QIR/Data-Types.md

```diff
@@ -2,15 +2,11 @@
 
 We define LLVM representations for a variety of classical and quantum data types.
 
-In most cases, we represent complex types as pointers to opaque LLVM structure
-types.
-This allows each target to provide a structure definition appropriate for that
-target.
+Representing the types used for qubits and measurement results as pointers to opaque LLVM structure types allows each target to provide a structure definition appropriate for that target.
 
-### Reference Counting
+### Reference and Access Counting
 
-In QIR, all types represented as pointers, other than qubits, are reference-counted.
-All types follow the same pattern:
+QIR specifies a set of runtime functions for types that are represented as pointers that may be used by the language specific compiler to expose them as immutable types in the language. 
 
 - Runtime routines that create a new instance always initialize the instance
```

Comment:
How the user count should be initialized?

Reply:
I'll add a sentence that the user count is supposed to be initialized to zero at the corresponding create runtime function.